service/build.gradle

```diff
@@ -72,7 +73,6 @@ dependencies {
     spotbugs "com.github.spotbugs:spotbugs:${spotbugs.toolVersion.get()}"
     spotbugs 'org.slf4j:slf4j-simple:2.0.7'
 
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.3'
```

Comment:
Spring Boot automatically pulls in JUnit; allowing it to do so means no chance of version conflicts here.